build/package.py

```diff
@@ -24,36 +30,29 @@ def main():
                       help='Path to application snapshot')
   parser.add_argument('--depfile', type=str, required=True,
                       help='Where to output dependency information')
-  parser.add_argument('--build-output', type=str, required=True,
-                      help='The final target to use in the dependency information')
-  parser.add_argument('--bundle', type=str, required=True,
+  parser.add_argument('--output-file', type=str, required=True,
                       help='Where to output application bundle')
-  parser.add_argument('--bundle-header', type=str, required=True,
-                      help='String to prepend to bundle')
 
   args = parser.parse_args()
 
+  env = os.environ.copy()
+  env['LD_LIBRARY_PATH'] = args.root
```

Comment:
It's so that the dart executable finds the shared libraries that were built for it in the Fuchsia tree. Otherwise the default search path finds another version of libssl.so which is missing some symbols the executable expects. This is the main reason why I'm doing this in a script rather than straight in the BUILD.gn file (which was my initial approach).